packages/charts/chart-utilities/src/PlotlySchemaConverter.ts

```diff
@@ -177,14 +177,36 @@ const validateBarData = (data: Partial<PlotData>) => {
 };
 
 const validateScatterData = (data: Partial<PlotData>) => {
-  if (
-    ['markers', 'text+markers', 'markers+text'].includes(data.mode ?? '') &&
-    !isNumberArray(data.x) &&
-    !isDateArray(data.x)
-  ) {
-    throw new Error(`${UNSUPPORTED_MSG_PREFIX} ${data.type}, mode: ${data.mode}, xAxisType: String`);
-  } else if (!isNumberArray(data.y) && !isStringArray(data.y)) {
-    throw new Error(`Non numeric or string Y values encountered.`);
+  const mode = data.mode ?? '';
+
+  if (['markers'].includes(mode)) {
+    const xAxisType = data && data.x && data.x.length > 0 ? typeof data?.x?.[0] : 'undefined';
+    const yAxisType = data && data.y && data.y.length > 0 ? typeof data?.y?.[0] : 'undefined';
+    // Any series having only markers -> Supported number x/string x/date x + number y
+    if (!isNumberArray(data.x) && !isStringArray(data.x) && !isDateArray(data.x)) {
+      throw new Error(`${UNSUPPORTED_MSG_PREFIX} ${data.type}, mode: ${mode}, xAxisType: ${xAxisType}`);
+    }
+    if (!isNumberArray(data.y)) {
+      throw new Error(`${UNSUPPORTED_MSG_PREFIX} ${data.type}, mode: ${mode}, yAxisType: ${yAxisType}`);
+    }
+  } else if (['lines+markers', 'markers+lines', 'text+lines+markers'].includes(mode)) {
```

Comment:
>else if [](http://example.com/codeflow?start=4&length=7)

we are ignoring markers in case of lines+markers and string x axis. Combine these 2 else if blocks